internal/authn/oidc/oidc_test.go

```diff
@@ -180,6 +188,27 @@ func TestRemoteOidcAuthenticator_Authenticate(t *testing.T) {
 			},
 			expectedError: "invalid subject",
 		},
+		{
+			testDescription: "when_the_subject_of_the_token_is_not_a_string,_MUST_return_'invalid_subject'_error",
+			testSetup: func() (*RemoteOidcAuthenticator, context.Context, error) {
+				return quickConfigSetup(
+					"kid_1",
+					"kid_1",
+					"right_issuer",
+					"right_audience",
+					nil,
+					[]string{"some-user"},
```

Comment:
```suggestion
					[]string{"valid-sub-1", "valid-sub-2"},
```

just for readability :)